backend/src/Notifo/Areas/Api/Controllers/Users/Dtos/ProfileDto.cs

```diff
@@ -64,6 +69,7 @@ public static ProfileDto FromDomainObject(User source, App app)
         {
             var result = SimpleMapper.Map(source, new ProfileDto());
 
+            result.AllowedTopics = source.AllowedTopics.ToArray();
```

Comment:
If you use the same type that is also used in the User class you can get rid of this line and also save a few allocations.